### ISSUE TITLE
[stacked_generator] unexpected import in app.locator.dart

### ISSUE THREAD

**Issue by katusiky** [Stacked-Org/stacked#289]
The `app.locator.dart` file is being generated with an empty import line.

```
// GENERATED CODE - DO NOT MODIFY BY HAND

// **************************************************************************
// StackedLocatorGenerator
// **************************************************************************

// ignore_for_file: public_member_api_docs

import 'package:stacked/stacked.dart';
import 'package:stacked_services/stacked_services.dart';

import ''; <--- THIS IS THE UNEXPECTED IMPORT
import '../services/authentication_service.dart';

final locator = StackedLocator.instance;

Future setupLocator() async {
  locator.registerLazySingleton(() => NavigationService());
  locator.registerLazySingleton(() => AuthenticationService());
}
```

**Comment by FilledStacks** [comment]
Thanks. Will get this removed.

**Comment by hurbes** [comment]
Working on it.

**Comment by FilledStacks** [comment]
Fix Pushed in 0.3.2
